docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
I would vote to rename this file to show that this only starts a kafka server, e.g. `docker-compose-kafka` in order to not let the user confuse it for a d3a docker-compose file.
Also, I see this only being used for the integrationtests. How about moving it into the integration_test folder?

Reply:
I'll move it in another directory since I removed the dependency of Kafka for the integration tests.